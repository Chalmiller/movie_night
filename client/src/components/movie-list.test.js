import React from 'react';
import { shallow } from 'enzyme';
import MovieList from './movie-list.component';

it('renders comment list without crashing', () => {
  expect(shallow(<MovieList/>)).toMatchSnapshot();
});