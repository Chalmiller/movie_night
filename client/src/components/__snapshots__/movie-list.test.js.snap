// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders comment list without crashing 1`] = `ShallowWrapper {}`;
